geodata/migrations/0004_auto_20200903_1811.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 3.1 on 2020-09-03 18:11
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('geodata', '0003_auto_20200818_1615'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='BaseImageFile',
+            fields=[
+                ('image_file_id', models.AutoField(primary_key=True, serialize=False)),
+            ],
+        ),
+        migrations.AddField(
+            model_name='imagefile',
+            name='baseimagefile_ptr',
+            field=models.OneToOneField(
+                auto_created=True,
+                default=0,
+                on_delete=django.db.models.deletion.CASCADE,
+                parent_link=True,
+                to='geodata.baseimagefile',
+            ),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name='imageentry',
+            name='image_file',
+            field=models.OneToOneField(
+                null=True, on_delete=django.db.models.deletion.CASCADE, to='geodata.baseimagefile'
+            ),
+        ),
```

Comment:
This might not be right... Since I am adding an inherited model, this might bot be the right migration to do.